--> executable follows <--
srun -n 1 python /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/albany_driver.py -mesh green-3-20km -ms1_smoother_name MT_Gauss_Seidel -ms1_relaxation_num_sweeps 1 -ms1_relaxation_damping_factor 1.069645 -ms1_relaxation_inner_damping_factor 0.980923 -ms1_chebyshev_degree 4 -ms1_chebyshev_eig_ratio 43.339622 -ms1_chebyshev_max_iters 69 -ms4_smoother_name Two_stage_Gauss_Seidel -ms4_relaxation_num_sweeps 1 -ms4_relaxation_damping_factor 0.824458 -ms4_relaxation_inner_damping_factor 1.064269 -ms4_chebyshev_degree 2 -ms4_chebyshev_eig_ratio 19.592675 -ms4_chebyshev_max_iters 73 -input /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/../../data -output /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/data/solvetime.3.txt -timelimit 500 || echo ""
--> end executable <--
Traceback (most recent call last):
  File "/global/homes/m/mcarlson/LandIce/builds/albany-tune-prlm-cuda-gcc-sfad12/install/lib/python3.9/site-packages/PyAlbany/Utils.py", line 8, in <module>
    from PyAlbany import Albany_Pybind11 as wpa
ImportError: libcupti.so.11.7: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/albany_driver.py", line 1, in <module>
    from PyAlbany import Utils
  File "/global/homes/m/mcarlson/LandIce/builds/albany-tune-prlm-cuda-gcc-sfad12/install/lib/python3.9/site-packages/PyAlbany/Utils.py", line 10, in <module>
    import Albany_Pybind11 as wpa
ModuleNotFoundError: No module named 'Albany_Pybind11'
srun: error: nid001544: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=3308387.3
--> executable follows <--
srun -n 1 python /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/albany_driver.py -mesh green-3-20km -ms1_smoother_name MT_Gauss_Seidel -ms1_relaxation_num_sweeps 1 -ms1_relaxation_damping_factor 1.069645 -ms1_relaxation_inner_damping_factor 0.980923 -ms1_chebyshev_degree 4 -ms1_chebyshev_eig_ratio 43.339622 -ms1_chebyshev_max_iters 69 -ms4_smoother_name Two_stage_Gauss_Seidel -ms4_relaxation_num_sweeps 1 -ms4_relaxation_damping_factor 0.824458 -ms4_relaxation_inner_damping_factor 1.064269 -ms4_chebyshev_degree 2 -ms4_chebyshev_eig_ratio 19.592675 -ms4_chebyshev_max_iters 73 -input /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/../../data -output /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/data/solvetime.3.txt -timelimit 500 || echo ""
--> end executable <--
Traceback (most recent call last):
  File "/global/homes/m/mcarlson/LandIce/builds/albany-tune-prlm-cuda-gcc-sfad12/install/lib/python3.9/site-packages/PyAlbany/Utils.py", line 8, in <module>
    from PyAlbany import Albany_Pybind11 as wpa
ImportError: libcupti.so.11.7: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/albany_driver.py", line 1, in <module>
    from PyAlbany import Utils
  File "/global/homes/m/mcarlson/LandIce/builds/albany-tune-prlm-cuda-gcc-sfad12/install/lib/python3.9/site-packages/PyAlbany/Utils.py", line 10, in <module>
    import Albany_Pybind11 as wpa
ModuleNotFoundError: No module named 'Albany_Pybind11'
srun: error: nid001544: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=3308387.13
--> executable follows <--
srun -n 1 python /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/albany_driver.py -mesh green-3-20km -ms1_smoother_name MT_Gauss_Seidel -ms1_relaxation_num_sweeps 1 -ms1_relaxation_damping_factor 1.069645 -ms1_relaxation_inner_damping_factor 0.980923 -ms1_chebyshev_degree 4 -ms1_chebyshev_eig_ratio 43.339622 -ms1_chebyshev_max_iters 69 -ms4_smoother_name Two_stage_Gauss_Seidel -ms4_relaxation_num_sweeps 1 -ms4_relaxation_damping_factor 0.824458 -ms4_relaxation_inner_damping_factor 1.064269 -ms4_chebyshev_degree 2 -ms4_chebyshev_eig_ratio 19.592675 -ms4_chebyshev_max_iters 73 -input /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/../../data -output /global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/data/solvetime.3.txt -timelimit 500 || echo ""
--> end executable <--
Traceback (most recent call last):
  File "/global/homes/m/mcarlson/LandIce/builds/albany-tune-prlm-cuda-gcc-sfad12/install/lib/python3.9/site-packages/PyAlbany/Utils.py", line 8, in <module>
    from PyAlbany import Albany_Pybind11 as wpa
ImportError: libcupti.so.11.7: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/global/homes/m/mcarlson/LandIce/ALITune/optimize/sampling/nrun10_check2/albany_driver.py", line 1, in <module>
    from PyAlbany import Utils
  File "/global/homes/m/mcarlson/LandIce/builds/albany-tune-prlm-cuda-gcc-sfad12/install/lib/python3.9/site-packages/PyAlbany/Utils.py", line 10, in <module>
    import Albany_Pybind11 as wpa
ModuleNotFoundError: No module named 'Albany_Pybind11'
srun: error: nid001544: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=3308387.23
